networking_private_dns_resolver_inbound_endpoints.tf

```
@@ -0,0 +1,28 @@
+
+module "private_dns_resolver_inbound_endpoints" {
+  source   = "./modules/networking/private_dns_resolvers_inbound_endpoints"
+  for_each = local.networking.private_dns_resolver_inbound_endpoints
+
+  global_settings         = local.global_settings
+  client_config           = local.client_config
+  settings                = each.value
+  private_dns_resolver_id = can(each.value.private_dns_resolver_id) ? each.value.private_dns_resolver_id : local.combined_objects_private_dns_resolvers[try(each.value.private_dns_resolver.lz_key, local.client_config.landingzone_key)][each.value.private_dns_resolver.key].id
```

To be consistent with other changes
```suggestion
  private_dns_resolver_id = can(each.value.private_dns_resolver_id) || can(each.value.private_dns_resolver.id) ? try(each.value.private_dns_resolver_id, each.value.private_dns_resolver.id) : local.combined_objects_private_dns_resolvers[try(each.value.private_dns_resolver.lz_key, local.client_config.landingzone_key)][each.value.private_dns_resolver.key].id
```